README.md

```diff
@@ -116,71 +87,19 @@ That's it! You are now running a standalone version of Parse Server on your mach
-
-Now that you're running Parse Server, it is time to save your first object. We'll use the [REST API](http://docs.parseplatform.org/rest/guide), but you can easily do the same using any of the [Parse SDKs](http://parseplatform.org/#sdks). Run the following:
-
-```bash
-$ curl -X POST \
--H "X-Parse-Application-Id: APPLICATION_ID" \
--H "Content-Type: application/json" \
--d '{"score":1337,"playerName":"Sean Plott","cheatMode":false}' \
-http://localhost:1337/parse/classes/GameScore
-```
-
-You should get a response similar to this:
+### Getting started with SDKs
 
-```js
-{
-  "objectId": "2ntvSpRGIK",
-  "createdAt": "2016-03-11T23:51:48.050Z"
-}
-```
-
-You can now retrieve this object directly (make sure to replace `2ntvSpRGIK` with the actual `objectId` you received when the object was created):
+Now that you're running Parse Server, it is time to save your first object. Parse Server has a broad range of [SDKs](http://parseplatform.org/#sdks) you can use to connect to your server.
 
-```bash
-$ curl -X GET \
-  -H "X-Parse-Application-Id: APPLICATION_ID" \
-  http://localhost:1337/parse/classes/GameScore/2ntvSpRGIK
-```
-```json
-// Response
-{
-  "objectId": "2ntvSpRGIK",
-  "score": 1337,
-  "playerName": "Sean Plott",
-  "cheatMode": false,
-  "updatedAt": "2016-03-11T23:51:48.050Z",
-  "createdAt": "2016-03-11T23:51:48.050Z"
-}
-```
-
-Keeping tracks of individual object ids is not ideal, however. In most cases you will want to run a query over the collection, like so:
-
-```bash
-$ curl -X GET \
-  -H "X-Parse-Application-Id: APPLICATION_ID" \
-  http://localhost:1337/parse/classes/GameScore
-```
-```json
-// The response will provide all the matching objects within the `results` array:
-{
-  "results": [
-    {
-      "objectId": "2ntvSpRGIK",
-      "score": 1337,
-      "playerName": "Sean Plott",
-      "cheatMode": false,
-      "updatedAt": "2016-03-11T23:51:48.050Z",
-      "createdAt": "2016-03-11T23:51:48.050Z"
-    }
-  ]
-}
-
-```
-
-To learn more about using saving and querying objects on Parse Server, check out the [Parse documentation](http://docs.parseplatform.org).
+- [Getting started with iOS](https://docs.parseplatform.org/ios/guide/#getting-started)
+- [Getting started with Android](https://docs.parseplatform.org/android/guide/#getting-started)
+- [Getting started with JavaScript](https://docs.parseplatform.org/js/guide/#getting-started)
+- [Getting started with .NET + Xamarin](https://docs.parseplatform.org/dotnet/guide/#getting-started)
+- [Getting started with macOS](https://docs.parseplatform.org/macos/guide/#getting-started)
+- [Getting started with Unity](https://docs.parseplatform.org/unity/guide/#getting-started)
+- [Getting started with PHP](https://docs.parseplatform.org/php/guide/#getting-started)
+- [Getting started with Arduino](https://docs.parseplatform.org/arduino/guide/#getting-started)
+- [Getting started with Embedded C](https://docs.parseplatform.org/embedded_c/guide/#getting-started)
```

Comment:
This is a bit less wordy & I think it's helpful to have a link to the REST and GraphQL guides here.

```suggestion
Getting started with:
- [REST API](https://docs.parseplatform.org/rest/guide/#your-configuration)
- [GraphQL API](https://docs.parseplatform.org/graphql/guide/#getting-started)
- [iOS SDK](https://docs.parseplatform.org/ios/guide/#getting-started)
- [Android SDK](https://docs.parseplatform.org/android/guide/#getting-started)
- [JavaScript SDK](https://docs.parseplatform.org/js/guide/#getting-started)
- [.NET + Xamarin SDK](https://docs.parseplatform.org/dotnet/guide/#getting-started)
- [macOS SDK](https://docs.parseplatform.org/macos/guide/#getting-started)
- [Unity SDK](https://docs.parseplatform.org/unity/guide/#getting-started)
- [PHP SDK](https://docs.parseplatform.org/php/guide/#getting-started)
- [Arduino SDK](https://docs.parseplatform.org/arduino/guide/#getting-started)
- [Embedded C SDK](https://docs.parseplatform.org/embedded_c/guide/#getting-started)
```